README.md

```diff
@@ -64,23 +64,32 @@ This project can be run in a [Docker](https://www.docker.com/) container.
 You need a local database instance listening on port 8080. Here is how to set that up:
 
 ``` sh
-docker run --name mysql -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 -d mysql
-export LEDGER_DB_URI=mysql://root:password@localhost/fivebells
-npm run migrate
+docker run --name db -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres
+export LEDGER_DB_URI=postgres://postgres:password@localhost
```

Comment:
I'd prefer if we got rid of the `-p 5432:5432`. Again, because it's likely to collide with another postgres server.

Instead, you can do:

``` sh
export LEDGER_DB_URI=postgres://postgres:password@`docker inspect --format '{{ .NetworkSettings.IPAddress }}' five-bells-ledger-db`
```